src/NuGet.Clients/NuGet.VisualStudio.Interop/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("NuGet.VisualStudio.Interop")]
+[assembly: AssemblyDescription("NuGet/Visual Studio interop assembly")]
+
+// The version of this assembly should never be changed.
+
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Do we need AssemblyVersion through ASsemblyTradermark here?
Shouldn't they be inherited?